### ISSUE TITLE
Allow to jump to specific node of the Lua-equivalent output

### ISSUE THREAD

**Issue by oblitzitate** [udayvir-singh/tangerine.nvim#9]
I'm not sure if it's difficult to implement, but It'd be nice to jump to the equivalent node at cursor position when running a command like `FnlGoToOutput`.

This would be useful for running proxy commands to the Lua LSP server. For instance, you create a command that jumps to the equivalent node in the Lua output, then runs the "go to definition" lsp command in the Lua output.

**Comment by udayvir-singh** [comment]
@oblitzitate

This feature is too difficult (if not near impossible) to implement because fennel doesn't provide any lsp or even a way to peak lua at all. Even if I was able to implement it, the project would get addition 500+ loc which would be counter-intuitive since this plugin was designed originally to only compile fennel files and load them. The eval/utility functions were added later on.

So I will be closing this issue since it is out of scope for this project.
